Linkta-org/core
# Issue #331 Ted/feature password confirmation

## Description
Implemented a confirm password field on the user signup page along with the corresponding error message if the passwords don't match.  Ticket #331 

Increased MAX output token form 1000 to 1500, as 1000 was not sufficient, this was intended to be a temporary fix, but I propose we leave it for now, until we change the limits for the response to come in under the max output value.


Fixes # (issue)
#331 
## Type of change







## How Can this be tested? Testing Plan to review this PR
To test that it stops the process of signing up if the password is not verified, use any sample email when filling out the form, and make sure to use two different password variations in the 'password' and 'confirm password field', you should see a note that requests you to try again because the passwords don't match.  
To ensure that it still creates new members, use a dummy email, and ensure the password is entered the same each time.





## Checklist:




## Additional Information:
